tests/resource_filtering_test.go

```diff
@@ -278,7 +286,14 @@ var _ = Describe("Resource includes", func() {
 
 		// Deleting the backup also deletes all restores, volume snapshots etc.
 		err := DeleteBackup(timeout, backupName, r.BackupNamespace)
-		Expect(err).ToNot(HaveOccurred())
+		if err != nil {
+			fmt.Fprintf(GinkgoWriter, "Err: %s\n", err)
+		}
+
+		err = client.CoreV1().Namespaces().Delete(context.TODO(), namespace.Name, metav1.DeleteOptions{})
+		if err != nil && !apierrs.IsNotFound(err) {
+			fmt.Fprintf(GinkgoWriter, "Err: %s\n", err)
+		}
```

Comment:
This is AfterEach, so I want all steps to run. DeleteBackup, DeleteNamespace, and then cancel if needed. I do not want it to stop on first error. 
If it fails on DelteBackup and I use `Expect(err).ToNot(HaveOccurred())` it will fail and exit, and then it will not remove the namespace in next instruction, which is even worse for other tests than just logging and going forward. 

Maybe, it would be even better to get a boolean and fail at the very end? I think It would complicate things too much. 

Anyway, If it fails here that it is some kind of environment problem and not a test failure. So I think

Reply:
I am planing to modify other `AfterEach` the same way If I find `Expect()` there.